### DISCUSSION TITLE
Show Batch details expanded by default, for a course

### DISCUSSION THREAD

**Discussion by mmanimegalai** [Sunbird-Ed/Community#33] [Enhancement request]
**Context:**

This is a DIKSHA request. 

On the Portal, if a user opens a Course, there is a “Batch Details” section. However as this is collapsed by default, user has to click on it to see the details. It has the information of Start date, end date and enrollment end date of the current batch. Since this has very important information related to - by when the user has to enroll + by when the user has to complete the course, 

1. This section has to be by default expanded at all times - before and after enrolling to the batch. 
2. This section should always be the first section on the right hand side before enrollment and the second section on the right hand side after enrollment. certificate section should be placed below the batch details

**Comment by luckynvdu** [comment]
Thanks @mmanimegalai  for sharing the requirements. We will be picking this in 4.10.2 as part of https://project-sunbird.atlassian.net/browse/SB-30473
@sharathkashyap

**Reply by sharathkashyap** [reply]
@mmanimegalai @luckynvdu Will be closing this thread as the followup can happen on Issue Tracker Specifically.
